subsys/testsuite/ztest/src/ztest.c

```diff
@@ -427,6 +427,9 @@ void end_report(void)
 	} else {
 		TC_END_REPORT(TC_PASS);
 	}
+#ifdef CONFIG_STM32_LPTIM_TIMER
```

Comment:
we should not be adding hardware related code into the test framework. Please find another way.

Reply:
yes, I understand,